Packet++/src/DhcpLayer.cpp

```diff
@@ -125,51 +125,50 @@ std::string DhcpLayer::toString() const
 	std::string msgType = "Unknown";
 	switch (getMesageType())
 	{
-	case DHCP_DISCOVER:
-	{
-		msgType = "Discover";
-		break;
-	}
-	case DHCP_OFFER:
-	{
-		msgType = "Offer";
-		break;
-	}
-	case DHCP_REQUEST:
-	{
-		msgType = "Request";
-		break;
-	}
-	case DHCP_DECLINE:
-	{
-		msgType = "Decline";
-		break;
-	}
-	case DHCP_ACK:
-	{
-		msgType = "Acknowledge";
-		break;
-	}
-	case DHCP_NAK:
-	{
-		msgType = "Negative Acknowledge";
-		break;
-	}
-	case DHCP_RELEASE:
-	{
-		msgType = "Release";
-		break;
-	}
-	case DHCP_INFORM:
-	{
-		msgType = "Inform";
-		break;
-	}
-	default:
-		break;
-
+		case DHCP_DISCOVER:
+		{
+			msgType = "Discover";
+			break;
+		}
+		case DHCP_OFFER:
+		{
+			msgType = "Offer";
+			break;
+		}
+		case DHCP_REQUEST:
+		{
+			msgType = "Request";
+			break;
+		}
+		case DHCP_DECLINE:
+		{
+			msgType = "Decline";
+			break;
+		}
+		case DHCP_ACK:
+		{
+			msgType = "Acknowledge";
+			break;
+		}
+		case DHCP_NAK:
+		{
+			msgType = "Negative Acknowledge";
+			break;
+		}
+		case DHCP_RELEASE:
+		{
+			msgType = "Release";
+			break;
+		}
+		case DHCP_INFORM:
+		{
+			msgType = "Inform";
+			break;
+		}
+		default:
+			break;
+	

```

Comment:
The existing `switch` indentation is the same throughout the project, I prefer not to change it